packages/validation/src/loadbalancers.schema.ts

```diff
@@ -155,36 +155,42 @@ export const UpdateRouteSchema = object({
 });
 
 export const UpdateConfigurationSchema = object({
-  label: string().min(1),
-  port: number(),
+  label: string().min(1, 'Label must not be empty.'),
+  port: number()
+    .min(0, 'Port must be greater than 0.')
```

Comment:
```suggestion
    .min(1, 'Port must be greater than 0.')
```
Currently, this allows a port number of 0.